packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -276,6 +278,20 @@ export async function publicSettings(
     // settings
     const configDoc = await configs.getSettingsConfigDoc()
     const config = configDoc.config
+
+    // License/Feature Checks
+    const license = await licensing.getLicense()
```

Comment:
Should use:

```ts
licensing.cache.getCachedLicense()
```

The `getLicense` will go to account portal and fetch the license from stripe, where are the above will use the local cache and populate if needed.